prow/config.yaml

```diff
@@ -1727,6 +1727,49 @@ postsubmits:
         - name: cache-ssd
           hostPath:
             path: /mnt/disks/ssd0
+    - name: ci-kubernetes-e2e-gce-1-6-1-7-master-skew
+      agent: kubernetes
+      spec:
+        containers:
+        - image: gcr.io/k8s-testimages/e2e-kubeadm:v20170927-71ecbe7f
```

Comment:
I bumped the kubeadm image a few days ago. Can you update your commit to use the new tag?

https://github.com/kubernetes/test-infra/blob/master/prow/config.yaml#L112